dbms/src/Interpreters/Settings.h

```diff
@@ -301,7 +301,11 @@ struct Settings
       *     It is for testing purposes, the syntax will likely change soon and the server will not be able \
       *     to load the tables created this way. You have been warned. \
       */ \
-    M(SettingBool, experimental_merge_tree_allow_custom_partitions, false)
+    M(SettingBool, experimental_merge_tree_allow_custom_partitions, false) \
+    /* Timeout for flushing data from streaming storages. */ \
+    M(SettingMilliseconds, stream_flush_interval_ms, DEFAULT_QUERY_LOG_FLUSH_INTERVAL_MILLISECONDS) \
+    /* Schema identifier (used by schema-based formats) */ \
+    M(SettingString, schema, "")
```

Comment:
There are no schema-based formats currently, but I'll be adding a support for some later and this ended up in an earlier commit. Let me know if you want me to remove the schema setting and add it in a different PR.

Reply:
Schema is very generic name. Probably, `format_schema` will be better.
Yes, better to add this setting only when it will be used.